apps/demos/Demos/DataGrid/CustomEditors/React/EmployeeDropDownBoxComponent.tsx

```diff
@@ -12,7 +12,7 @@ const dropDownOptions = { width: 500 };
 const ownerLabel = { 'aria-label': 'Owner' };
 
 const EmployeeDropDownBoxComponent = (props) => {
-  const [selectedRowKeys, setSelectedRowKeys] = useState([props.data.value]);
+  const [selectedRowKeys, setSelectedRowKeys] = useState(() => (props.data.value !== null && props.data.value !== undefined ? [props.data.value] : []));
```

Comment:
Let's move on the initial state value to a separate variable:

```suggestion
  const { data: { value: dataValue } } = props;
  const initialSelectedRowKeys = dataValue !== null && dataValue !== undefined ? [dataValue] : []
  const [selectedRowKeys, setSelectedRowKeys] = useState(initialSelectedRowKeys);
```